generated/nixnetsocket/nixnetsocket_service.cpp

```diff
@@ -297,7 +370,7 @@ namespace nixnetsocket_grpc {
       auto socket_grpc_session = request->socket();
       nxSOCKET socket = session_repository_->access_session(socket_grpc_session.id(), socket_grpc_session.name());
       auto addr = allocate_output_storage<nxsockaddr, SockAddr>();
-      nxsocklen_t addrlen {};
+      auto addrlen = static_cast<nxsocklen_t>(sizeof(addr.storage));
```

Comment:
Annoying subtle change: `addrlen` is an in/out parameter. If we don't initialize it, the driver will error out because it thinks we didn't give it space for an address. I'm solving this by enabling "hardcoded" on output parameters. We're already setting up the max required storage size, we just need to tell the driver that.